setup.py

```diff
@@ -61,7 +61,7 @@ def run(self):
 
 setup(
     name="dask_sql",
-    description="Dask SQL",
+    description="SQL query layer for Dask",
     url="http://github.com/dask-contrib/dask-sql/",
```

Comment:
Also wondering if this should be `https`

```suggestion
    url="https://github.com/dask-contrib/dask-sql/",
```